docs/advanced/gcp-lb.md

```diff
@@ -1,5 +1,7 @@
 # GCP Load Balancers for type=LoadBalacer of Kubernetes Services
 
+> **Deprecated**: Since v1.31 (the Kubespray counterpart is v2.27), Kubernetes no longer supports `cloud_provider`. (except external cloud provider)
```

Comment:
Instead of a notice, we should remove the mention  of --cloud-provider=gce below and similar refs.
Not sure if we should keep that notice or not, but if we do, that should be `**Removed**` instead of `**Deprecated**`

Reply:
I thought it would be better to remove it, but I wasn't sure how many people were using it in the past with 2.25 and 2.26 (even though Kubernetes had already announced that it would be deprecated), so I opted to keep it for now.

I will change it to `**Removed**` later.